targets/targets.json

```diff
@@ -2446,7 +2453,8 @@
         "device_has_add": [
             "SERIAL_ASYNCH",
             "FLASH",
-            "MPU"
+            "MPU",
+            "WATCHDOG"
```

Comment:
This is not needed as target inherits from FAMILY_STM32 where WATCHDOG is already defined